crates/networking/p2p/sync.rs

```diff
@@ -72,8 +79,10 @@ pub enum SyncMode {
 /// Manager in charge the sync process
 /// Only performs full-sync but will also be in charge of snap-sync in the future
 #[derive(Debug)]
-pub struct SyncManager {
-    sync_mode: SyncMode,
+pub struct Syncer {
+    /// This is also held by the SyncSupervidor allowing it to track the latest syncmode, without modifying it
```

Comment:
```suggestion
    /// This is also held by the SyncSupervisor allowing it to track the latest syncmode, without modifying it
```

Reply:
Nice catch! It also looks to be outdated, Ill update it